mappings/net/minecraft/entity/ai/pathing/PathingCoordinator.mapping

```diff
@@ -20,6 +28,17 @@ CLASS none/vx
 		ARG 5 z
 		ARG 6 volume
 		ARG 7 pitch
+	METHOD a didBoundingBoxChange (Lnone/aiw;Lnone/cn;Lnone/asm;Lnone/asm;)Z
```

Comment:
`hasBoundingBoxChanged` ? (Even though we use `has` prefix for boolean stuff usually `^^`)